tcms/wsgi.py

```diff
@@ -33,18 +33,13 @@
 # add tcms's core lib path
 sys.path.append(os.path.join(tcms.__path__[0], 'core', 'lib'))
 
-# This application object is used by any WSGI server configured to use this
-# file. This includes Django's development server, if the WSGI_APPLICATION
-# setting points here.
-_application = get_wsgi_application()
```

Comment:
Drop this commit for now. After further review I think we should keep the _application variable as it seems to provide some sort of caching local to the module. get_wsgi_application() does setup a Django instance which may not be desirable every time. However I'm not sure how the application() function works with Django nowadays.